src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -368,6 +368,9 @@ public override TypeWithAnnotations ReturnTypeWithAnnotations
             }
         }
 
+        public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations
+            => _property.GetterValueFlowAnalysisAnnotations;
```

Comment:
>_property.GetterValueFlowAnalysisAnnotations; [](start = 15, length = 45)

How do we know that this is a getter?

Reply:
> How do we know that this is a getter?

It doesn't look like this issue is addressed. The accessor can represent a setter, in that case the only appropriate return value is ```FlowAnalysisAnnotations.None``` regardless of the attributes on the property. We also shouldn't emit any attributes the setter's return type. 

---
In reply to: [295421641](https://github.com/dotnet/roslyn/pull/36444#discussion_r295421641) [](ancestors = 295421641)